js/zoninator.js

```diff
@@ -173,104 +174,121 @@ var zoninator = {}
 		var $list = $(returnData.content);
 		$('#zone-post-latest').html($list);
 
-	}
+	};
 
 	zoninator.addPost = function(postId) {
-		
+
 		zoninator.$zonePostSearch.trigger('loading.start');
-		
-		zoninator.ajax('add_post', {
-			zone_id: zoninator.getZoneId()
-			, post_id: postId
-		}, zoninator.addPostSuccessCallback);
-		
-	}
-	
+
+		zoninator.ajax('/zones/' + zoninator.getZoneId() + '/posts',
+			'POST', {post_id: postId}, zoninator.addPostSuccessCallback);
+
+	};
+
+	zoninator.ajax = function (endpoint, method, data, successCallback, errorCallback) {
+		if (!data) {
+			data = {};
+		}
+
+		zoninator.$zonePostSearch.trigger( 'zoninator.ajax', [ endpoint, method, data ] );

```

Comment:
This could be a back-compat issue as well as the original trigger allowed sites to customize the params sent via the AJAX endpoint (e.g. custom search params). Any thoughts on how we could handle this?